dagql/idtui/frontend.go

```diff
@@ -592,8 +592,10 @@ func (fe *Frontend) renderRow(out *termenv.Output, row *TraceRow, depth int) err
 	}
 	if !row.Span.Encapsulate || row.Span.Status().Code == codes.Error || fe.Verbosity >= 2 {
 		for _, child := range row.Children {
-			if err := fe.renderRow(out, child, depth); err != nil {
-				return err
+			if !child.Span.Encapsulated || row.Span.Status().Code == codes.Error || fe.Verbosity >= 2 {
```

Comment:
Ah ha, got it - so we'll reveal inner encapsulated spans if the parent fails. Nice.